XenAdmin/TabPages/ManageUpdatesPage.cs

```diff
@@ -502,8 +516,7 @@ into g
         private void DeleteAllAllertAction_Completed(ActionBase sender)
         {
             Program.Invoke(Program.MainWindow, () =>
-            {                
-                Rebuild();

```

Comment:
@letsboogey I've been looking at restoring the message on dismissal, however I'm wondering now whether it's correct to show it: the user has requested dismissal and not a check, so isn't it confusing to show immediately 'No updates found'? Maybe it's ok to have an empty grid and show the message later if the user requests a new check or any other action that requires a refresh?